Readme.md

```diff
@@ -553,3 +617,212 @@ Example:
 
 Inserts a no$gba debug message as described by GBATEK.
 
+# 5. Macros
```

Comment:
This section should probably be updated with the new macros. In general, I would avoid copying text from the old readme without checking that they are up to date first.